src/entities/uniqueAssets.ts

```diff
@@ -1,5 +1,61 @@
+import { AssetContract, AssetType } from '.';
+
+interface UniqueAssetLastSale {
+  total_price: string;
+  payment_token?: {
+    symbol: string;
+    usd_price: string;
+  };
+}
+
 export interface UniqueAsset {
```

Comment:
The following field changes/additions are all based on `parseAccountUniqueTokens` in `src/parsers/uniqueTokens`.

The only place this interface was used in the past was `buildRainbowUrl` in `src/utils`, and now it's used in the `uniqueTokens` reducer as well.